src/Microsoft.Tye.Core/Serialization/ConfigServiceParser.cs

```diff
@@ -393,6 +400,15 @@ private static void HandleBuildProperties(YamlSequenceNode yamlSequenceNode, Lis
                 buildProperties.Add(buildProperty);
             }
         }
+        private static void HandleDockerFileArgs(YamlSequenceNode yamlSequenceNode, Dictionary<string, string> dockerArguments)
+        {
+            foreach (var child in yamlSequenceNode.Children)
+            {
+                YamlParser.ThrowIfNotYamlMapping(child);
+                var keyValue = HandleServiceDockerArgsNameMapping((YamlMappingNode)child);
```

Comment:
nit: seems odd to return a key value pair here when you could just add the pair in HandleServiceDockerArgsNameMapping.

Reply:
refactored